assignment/scripts/loops-practice.js

```diff
@@ -13,17 +13,29 @@ for (let i=0; i<4; i++) {
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
+for (let i=0; i<6; i+= 1) {
```

Comment:
Stop me if you've heard this one, but I love space around my operators.
```suggestion
for (let i = 0; i < 6; i += 1) {
```